app/views/main.html

```diff
@@ -32,7 +32,7 @@
   <option value="3">Slide 4</option>
 </select>
 
-<simple-slider style="width:612px; height:612px" auto-play="false" change="{{actualslide}}">
+<simple-slider simple-slider-slide="actualslide" style="width:612px; height:612px" auto-play="false">
```

Comment:
I would like to see a better attribute name for the slide attribute, `actual-slide` or `current-slide` are definitely better